lib/modules/dosomething/dosomething_kudos/dosomething_kudos.module

```diff
@@ -307,5 +307,24 @@ function dosomething_kudos_term_is_selected($reportback_item, $term) {
  * @return bool
  */
 function dosomething_kudos_disable_reactions($nid) {
-  return !variable_get('dosomething_reportback_kudos_enabled', FALSE) || dosomething_helpers_get_variable('node', $nid, 'disable_reactions');
```

Comment:
Did we need this as an `||` statement?

also, remove the 👾 after the `return`.